applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_3D2N.cpp

```diff
@@ -1188,6 +1188,15 @@ CrBeamElement3D2N::GetCurrentNodalPosition() const
     return current_nodal_position;
 }
 
+void CrBeamElement3D2N::Calculate(const Variable<Matrix>& rVariable, Matrix& rOutput, const ProcessInfo& rCurrentProcessInfo)
+{
+    if (rVariable == LOCAL_ELEMENT_ORIENTATION)
+    {
+        rOutput.resize(msElementSize, msElementSize, false);
+        noalias(rOutput) = GetTransformationMatrixGlobal();

```

Comment:
@KlausBSautter we should speak how to make this more efficient => the fct `GetTransformationMatrixGlobal` should get the matrix as an input and not return it